pkg/decode/decode.go

```diff
@@ -27,6 +35,12 @@ func FromYaml(v interface{}, reader io.Reader) error {
 	return nil
 }
 
+// FromJSONWithYamlFallback reads an asyncapi document from an input in JSON format,
+// if this operation fails, the function tries to read an asyncapi document in yaml format.
+// If any decoding attempt succeeds, the result is stored in the value.
+// If both decoding attempts fail, function returns an error.
+//
+// See InvalidProperty, InvalidDocument, UnsupportedAsyncapiVersion in pkg error.
```

Comment:
Can we use / and write "in /pkg/error or it is forbidden in this kind of descriptions?